<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEditorDialog</class>
 <widget class="QDialog" name="BookEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>书籍信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelBookId">
       <property name="text">
        <string>书号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditBookId"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBookName">
       <property name="text">
        <string>书名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditBookName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAuthor">
       <property name="text">
        <string>作者：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditAuthor"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPublisher">
       <property name="text">
        <string>出版社：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPublisher"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPublishDate">
       <property name="text">
        <string>出版日期：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEditPublishDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelStock">
       <property name="text">
        <string>库存数量：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBoxStock">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxStatus"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelRemark">
       <property name="text">
        <string>备注：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="plainTextEditRemark">
       <property name="maximumHeight">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSubmit">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
